xrdp/xrdp_mm.c

```diff
@@ -2165,17 +2165,98 @@ xrdp_mm_check_chan(struct xrdp_mm *self)
 }
 
 /*****************************************************************************/
-int APP_CC
-xrdp_mm_check_wait_objs(struct xrdp_mm *self)
+static int APP_CC
+xrdp_mm_update_module_frame_ack(struct xrdp_mm *self)
+{
+    int fif;
+    struct xrdp_encoder *encoder;
+
+    encoder = self->encoder;
+    fif = encoder->frames_in_flight;
```

Comment:
Thanks for narrating your thought process but this is your opinion.  I find separate declaration easier to read.